example-dataset/pbmc3k-annotations-cxg.csv

```diff
@@ -0,0 +1,2641 @@
+# Annotations generated on 2020-02-22T12:26:07 using cellxgene version 0.14.0
```

Comment:
If this file is solely for testing (which I think it is), it should go in server/tests/test_datasets

I think same is true for the `.cxg` file in this directory

Reply:
Oh, pardon. I meant to remove this file. I can move the other files.